website/project/model.py

```diff
@@ -264,6 +277,13 @@ def create(cls, auth, **kwargs):
             save=False,
         )
 
+        if (comment.new_mentions):
+            comment.new_mentions = [mention for mention in comment.new_mentions if mention not in comment.old_mentions and validate_contributor(mention, comment.node.contributors)]
```

Comment:
Let's try to avoid long lines like this. Try abbreviating `mention` to `m` in the list comprehension, and if it's still longer than 100 chars (I haven't seen a local style guide on this, but usually 80 or maybe 100 are the standard) then maybe factor out the condition into a lambda:

```python
foo = lambda a: a or b or c
bar = [x for x in bar if foo(x)]
```

[`filter`](https://docs.python.org/2/library/functions.html#filter) is arguably more concise than a list comprehension here:

```python
bar = filter(foo, bar)
```

Some ideas. :-)